go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -27,25 +29,42 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
 )
 
+// SwitchNames contains the pair of possible switch related to a pod
+type SwitchNames struct {
```

Comment:
I am not sure about this, originaly the names where "subnetSwitch" and "currentSwitch" but after some reviews everyone was ok to replace it with "original" and "current" switch at the end is what is happening, the fact that we  use the switch names for the IP pool should not be reflected here I think.

Reply:
@maiqueb what do you think about @jcaamano proposal ?

Reply:
Honestly, I can't derive any meaning from those names (getSubnet => original switch name; getSwitch => current switch name).

IMO, it is preferable to be explicit.

Having said that, I'm not a maintainer.

Reply:
Why do we care about the original switch? 

If the only reason is because that's how we name the IP pool its IPs were allocated from, why not refer to that directly? Do we care about the original switch for any other reason?

That would prevent further questions on `why do you use original switch name instead of the current switch name here?` that have already happened.

This comment just tries to bring clarity of what the situation is really about: This pod is attached to this switch but got its IPs from this other pool.

Was just a suggestion though. I think at least the structure could be enriched with doc explaining why we care about the original switch name.

Reply:
I will add more comments to the struct.